src/index.html

```diff
@@ -7,21 +7,409 @@
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    />
     <link
       rel="stylesheet"
       href="styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo"
+          class="img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="block">
+        <div
+          class="block__card"
+          data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="imac"
+          />
+          <p class="block__apple">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="block__product">Product code: 195434</p>
+          <div class="block__stars">
+            <div class="stars stars--4">
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+            </div>
+            <p class="block__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="block__price">
+            <div class="block__price-txt">Price:</div>
```

Comment:
You need to use the semantic tag for text, fix it everywhere
```suggestion
            <p class="block__price-txt">Price:</p>
```